bin/onlineBuild.sh

```diff
@@ -46,13 +46,6 @@ echo ""
 
 sudo -Hu cool bash -c "./configure --enable-silent-rules --with-lokit-path=${cool_dir}/bundled/include --with-lo-path=${lo_dir}/instdir --with-max-connections=$cool_maxcon --with-max-documents=$cool_maxdoc --with-poco-includes=/usr/local/include --with-poco-libs=/usr/local/lib ${cool_configure_opts}" || exit 4
 # coolwsd+loleaflet take around 8.5/${cpu} minutes to compile on fast cpu
-cd ${cool_dir}/browser
```

Comment:
Why is this deleted?

Reply:
i previously added this code due to npm compilation errors. in the latest version of "cool " everything compiles fine without it.